packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -1,6 +1,4 @@
-const { Command } = require('@oclif/command');
-const { stdtermwidth } = require('@oclif/help/lib/screen');
-const { renderList } = require('@oclif/help/lib/list');
```

Comment:
`renderList` has been moved from `@oclif/help` to `@oclif/core`, but it can't be imported anymore.